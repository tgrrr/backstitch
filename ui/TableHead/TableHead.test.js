import React from 'react'
import renderer from 'react-test-renderer'
import { TableHead } from '.'

it('TableHead: default', () => {
    const component = renderer.create(
        <TableHead>
            <>
                TableHead
            </>
        </TableHead>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
