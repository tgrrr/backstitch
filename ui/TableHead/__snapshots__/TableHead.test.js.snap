// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableHead: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Tablehead "
    data-testid="Tablehead"
  >
    <thead
      className="MuiTableHead-root"
      role={null}
    >
      TableHead
    </thead>
  </div>
</div>
`;
